discovery_engine/test/integration/utils/local_newsapi_server.dart

```diff
@@ -69,14 +69,14 @@ class LocalNewsApiServer {
           break;
         case ReplyWith.data:
           switch (request.uri.path) {
-            case '/newscatcher/v1/search-news':
+            case '/newscatcher/v2/search':
             case '/_mlt':
               await _replyWithData(
                 request,
                 _snFiles.asMap()[_snCounter++] ?? 'search-news-empty.json',
               );
               break;
-            case '/newscatcher/v1/latest-headlines':
+            case '/newscatcher/v2/latest_headlines':
             case '/newscatcher/v2/trusted-sources':
```

Comment:
I think we should stick to either snake_case or kebab-case for the endpoint names, but it's probably something to configure on the proxy level as well, right?

Reply:
yes, the proxy routes have to be changed first. this is somewhat covered by the other tasks of the story.